Changes

```diff
@@ -122,6 +122,10 @@ Working version
 - #10605: manual, name few css classes to ease styling and maintainability.
   (Florian Angeletti, review by Wiktor Kuchta and Gabriel Scherer)
 
+- #8697, #10666: add M, m, n options of the OCAMLRUNPARAM to manual and man page
+  for ocamlrun command line options
+  (Dong An and Anukriti Kumar, review by David Allsopp, Gabriel Scherer)
+
```

Comment:
Sorry for the nitpick, the entry wants to be above the previous (ordering by the original issue number, rather than the PR). GitHub diff makes it look, um, interesting:
```suggestion
- #8697, #10666: add M, m, n options of the OCAMLRUNPARAM to manual and man page
  for ocamlrun command line options
  (Dong An and Anukriti Kumar, review by David Allsopp, Gabriel Scherer)

- #10605: manual, name few css classes to ease styling and maintainability.
  (Florian Angeletti, review by Wiktor Kuchta and Gabriel Scherer)
```